src/array_utils.rs

```diff
@@ -1,6 +1,8 @@
 use crate::Complex;
 use crate::FftNum;
-use std::ops::{Deref, DerefMut};
+use core::ops::{Deref, DerefMut};
+#[allow(unused_imports)]
```

Comment:
Why is this necessary? Can't you feature gate the import?

Reply:
Yeah, my bad. Fixed in [a6f4c2e](https://github.com/ejmahler/RustFFT/pull/124/commits/a6f4c2e1e583d1dc5ed6dd5ed1680814501897bb).